Resources/views/Datagrid/cells.html.twig

```diff
@@ -65,25 +65,25 @@
 
 {% block simple_cell -%}
     {% if data.value != null %}
-        <td>
-            {% if data.value is iterable %}
-                {% for value in data.value %}
-                    {% if value is not iterable %}
-                        {{ value }}
-                    {% endif %}
-                {% endfor %}
-            {% else %}
-                {% if data.property in ['title', 'name', 'username'] %}
-                    <a href="{{ path('opifer.crud.edit', {'slug': slug, 'id': row.id}) }}">
-                        {{ data.value }}
-                    </a>
-                {% elseif data.property in ['email'] %}
+        {% if data.value is iterable %}
+            {% for value in data.value %}
+                {% if value is not iterable %}
+                    {{ value }}
+                {% endif %}
+            {% endfor %}
+        {% else %}
+            {% if data.property in ['title', 'name', 'username'] %}
+                {{- block('link_cell') -}}
+            {% elseif data.property in ['email'] %}
+                <td>
```

Comment:
Is there a reason for moving the `<td>`? All cell data needs to be in a `<td>`, so I don't see why you'd move it...